docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -84,10 +82,18 @@ The change to the port number was made because of the need to provide a good usa
 
 There are two ways to respond to this breaking change:
 
-- Recommended: Explicitly set the `ASPNETCORE_HTTP_PORTS`, `ASPNETCORE_HTTPS_PORTS`, and `ASPNETCORE_URLS` environment variables to the desired port. Example: `docker run --rm -it -p 9999:80 -e ASPNETCORE_HTTP_PORTS=80 <my-app>`
+- (Recommended action) Explicitly set the `ASPNETCORE_HTTP_PORTS`, `ASPNETCORE_HTTPS_PORTS`, and `ASPNETCORE_URLS` environment variables to the desired port. Example: `docker run --rm -it -p 9999:80 -e ASPNETCORE_HTTP_PORTS=80 <my-app>`
```

Comment:
FYI: All of my instructions use `8000:blah` as the mapping. I have never seen `9999` used as an example. The number is arbitrary however I think a more standard number would encourage folks to not think our tech is special.